src/Components/Aspire.Azure.AppConfiguration/README.md

```diff
@@ -0,0 +1,162 @@
+}
+```
+
+#### Use feature flags
+
+To use feature flags, install the Feature Management library:
+
+```dotnetcli
+dotnet add package Microsoft.FeatureManagement
+```
+
+App Configuration will not load feature flags by default. To load feature flags, you can pass the `Action<AzureAppConfigurationOptions> configureOptions` delegate when calling `builder.AddAzureAppConfiguration`.
+
+```csharp
+builder.AddAzureAppConfiguration(
+    "appConfig",
+    configureOptions: options => options.UseFeatureFlags());
+
+// Register feature management services
+builder.Services.AddFeatureManagement();
+```
+
+You can then use `IVariantFeatureManager` to evaluate feature flags in your application:
+
+```csharp
+private readonly IVariantFeatureManager _featureManager;
+
+public MyController(IVariantFeatureManager featureManager)
+{
+    _featureManager = featureManager;
+}
+
+[HttpGet]
+public async Task<IActionResult> Get()
+{
+    if (await _featureManager.IsEnabledAsync("NewFeature"))
+    {
+        return Ok("New feature is enabled!");
+    }
+
+    return Ok("Using standard implementation.");
+}
+```
+
+For information about using the Feature Management library, please go to the [documentation](https://learn.microsoft.com/azure/azure-app-configuration/feature-management-dotnet-reference).
+
+## Configuration
+
+The .NET Aspire Azure App Configuration library provides multiple options to configure the Azure App Configuration connection based on the requirements and conventions of your project. Note that the App Config endpoint is required to be supplied, either in `AzureAppConfigurationSettings.Endpoint` or using a connection string.
+
+### Use a connection string
+
+When using a connection string from the `ConnectionStrings` configuration section, you can provide the name of the connection string when calling `builder.AddAzureAppConfiguration()`:
+
+```csharp
+builder.AddAzureAppConfiguration("appConfigConnectionName");
+```
+
+And then the App Configuration endpoint will be retrieved from the `ConnectionStrings` configuration section. The App Configuration store URI works with the `AzureAppConfigurationSettings.Credential` property to establish a connection. If no credential is configured, the [DefaultAzureCredential](https://learn.microsoft.com/dotnet/api/azure.identity.defaultazurecredential) is used.
+
+```json
+{
+  "ConnectionStrings": {
+    "appConfigConnectionName": "https://{store_name}.azconfig.io"
+  }
+}
+```
+
+### Use configuration providers
+
+The .NET Aspire Azure App Configuration library supports [Microsoft.Extensions.Configuration](https://learn.microsoft.com/dotnet/api/microsoft.extensions.configuration). It loads the `AzureAppConfigurationSettings` from configuration by using the `Aspire:Azure:AppConfiguration` key. Example `appsettings.json` that configures some of the options:
+
+```json
+{
+  "Aspire": {
+    "Azure": {
+      "AppConfiguration": {
+        "Endpoint": "http://YOUR_URI"
```

Comment:
We never use "http". You can have something like `"Endpoint": "YOUR_APPCONFIGURATION_ENDPOINT_URI"`.